nixops/backends/libvirtd.py

```diff
@@ -8,11 +8,17 @@
 import string
 import subprocess
 import time
+import libvirt
 
 from nixops.backends import MachineDefinition, MachineState
 import nixops.known_hosts
 import nixops.util
 
+# to prevent libvirt errors from appearin on screen, see
```

Comment:
appearin -> appearing

Reply:
fixed, I remove the example as one can just check out the link.